python/docs/source/getting_started/ps_10mins.ipynb

```diff
@@ -44,7 +44,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "s = ks.Series([1, 3, 5, np.nan, 6, 8])"
+    "s = ps.Series([1, 3, 5, np.nan, 6, 8])"
```

Comment:
@itholic, can you revert the changes in ipynb files? I will fix it in https://github.com/apache/spark/pull/33041

Reply:
Sure, just revert the changes